docs/setup-robusta/alertsui.rst

```diff
@@ -76,47 +46,28 @@ Choose the appropriate instructions below, based on whether you use the Promethe
 
          kubectl get crd | grep prometheus
 
-      To make sure Prometheus picks up Robusta's rule files and avoid duplication, add the following to the Kube Prometheus Stack configuration:
+      To make sure Prometheus picks up Robusta's rule files, add the following to the Kube Prometheus Stack configuration:
 
       .. code-block:: yaml
 
         prometheus: # collect rules from all namespaces and ignore label filters
             ruleNamespaceSelector: {}
             ruleSelector: {}
             ruleSelectorNilUsesHelmValues: false
-        defaultRules: # those rules are now managed by Robusta
-            rules:
-              alertmanager: false
-              etcd: false
-              configReloaders: false
-              general: false
-              kubeApiserverSlos: false
-              kubeControllerManager: false
-              kubeProxy: false
-              kubernetesApps: false
-              kubernetesResources: false
-              kubernetesStorage: false
-              kubernetesSystem: false
-              kubeSchedulerAlerting: false
-              kubeStateMetrics: false
-              network: false
-              nodeExporterAlerting: false
-              prometheus: false
-              prometheusOperator: false
-
-      Finally, to start syncing alerts to your cluster, add the following snippet to Robusta’s Helm values file named ``generated_values.yaml``:
+
+      Finally, make sure that the following snippet is in Robusta’s Helm values file named ``generated_values.yaml``:
 
       .. code-block:: yaml
 
-        enabledManagedConfiguration: true # Enable managed alerts
+        enabledManagedConfiguration: true # Enables managed alerts
 
       Then perform a :ref:`Helm Upgrade <Simple Upgrade>`.
 
 
 Disabling the Feature
 ---------------------------------
 
-If you choose to stop using the Robusta Alerts UI, you can do so at any time and go back to using built in Kube Prometheus Stack alerts. To do this, remove the config added in the previous step from your ``generated_values.yaml`` and do a :ref:`Helm Upgrade <Simple Upgrade>`.
+If you choose to stop using the Robusta Alerts UI, you can do so at any time and go back to using built in Kube Prometheus Stack alerts. To do this, remove the config added in the previous step from your ``generated_values.yaml``, set ``enabledManagedConfiguration: false`` and do a :ref:`Helm Upgrade <Simple Upgrade>`.
```

Comment:
Is this still true? `and go back to using built in Kube Prometheus Stack alerts` aren't the all idea of this change is that we aren't managing the builtin alerts any more? @Sheeproid WDYT?